docs/resources/server/enchantments.md

```diff
@@ -0,0 +1,85 @@
+# Enchantments
+
+Enchantments are enchancements that can be applied to tools and other items. As of 1.21, enchantments are stored on items as [Data Components], are defined in JSON, and are comprised of Enchantment Components.
+
+A new enchantment can be added by creating a JSON file in your namespace's `enchantment` datapack subfolder. For example, to create an enchantment called `examplemod:example_enchant`, one could create the file `data/examplemod/enchantment/example_enchantment.json`. Enchantments added this way are automatically registered into the game as part of the data pack loading process.
```

Comment:
Enchantments can only be added in this manner. It would probably be best to rephrase that last sentence.